source/byte_buf.c

```diff
@@ -394,7 +394,10 @@ bool aws_array_eq_c_str(const void *const array, const size_t array_len, const c
     return str_bytes[array_len] == '\0';
 }
 
-uint64_t aws_hash_array_ignore_case(const void *array, size_t len) {
+#pragma CPROVER check push
+#pragma CPROVER check disable "unsigned-overflow"
```

Comment:
I'm wondering whether that's scoped too wide? Maybe just put that around the affected lines rather than all the code in this function?

Reply:
Done.